### ISSUE TITLE
mandarin training

### ISSUE THREAD

**Issue by HandsLing** [NVIDIA/tacotron2#385]
i used tacotron2 to train a mandarin model from scratch, here is my hparams:
 max_wav_value=32768.0,
 sampling_rate=16000,
 filter_length=1024,
 hop_length=256,
 win_length=1024,
 n_mel_channels=80,
mel_fmin=0.0,
mel_fmax=8000.0,
other params was not changed, my symbols wav:
_pad        = '_'
_punctuation = '!\'(),.:;? '
_letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
mandarin = '0123456789'

symbols = [_pad] + list(mandarin) + list(_punctuation) + list(_letters)

after training of 23K steps, here is my aliment.png
![图片](https://user-images.githubusercontent.com/23291932/86564923-8e81e380-bf99-11ea-9c4a-d254524b68de.png)
and my loss was declined to 0.25, anyone can give me some advices? thx a lot

**Comment by HandsLing** [comment]
by the way, when i inference, a warning said: Warning! Reached max decoder steps , and the result wav was nothing but noise

**Comment by aishweta** [comment]
@HandsLing you need to change symbols in symbols.py , as you're using mandarin language

**Comment by HandsLing** [comment]
> 
> 
> @HandsLing you need to change symbols in symbols.py , as you're using mandarin language

Yes, I have converted Chinese characters into Pinyin，and the symbols have all symbols in my dataset

**Comment by cicido** [comment]
change the "english_cleaner" to "basic_cleaner"

**Comment by cicido** [comment]
> by the way, when i inference, a warning said: Warning! Reached max decoder steps , and the result wav was nothing but noise

change the max_decoder_steps=1000 to a larger number

**Comment by HandsLing** [comment]
> 
> 
> change the "english_cleaner" to "basic_cleaner"

i did it too

**Comment by cicido** [comment]
> > @HandsLing you need to change symbols in symbols.py , as you're using mandarin language
> 
> Yes, I have converted Chinese characters into Pinyin，and the symbols have all symbols in my dataset
number 1-5 should be in your sysbols

**Comment by cicido** [comment]
I think there may be somthing wrong in this repo and I have't get any meaningful result in mandarin speech synthsis

**Comment by cicido** [comment]
I have try some audio parameters in hparams.py many times,  and the alignments model does not work well

**Comment by HandsLing** [comment]
> 
> 
> I think there may be somthing wrong in this repo and I have't get any meaningful result in mandarin speech synthsis

i saw that someone succeed training mandarin with this repo

**Comment by wsywsywsywsywsy979** [comment]
> > I think there may be somthing wrong in this repo and I have't get any meaningful result in mandarin speech synthsis
> 
> i saw that someone succeed training mandarin with this repo

hello，do you have succeed? could you give me some advice or remind me what need to pay attention to or fix?
